### ISSUE TITLE
tracking: Handling of age-0, age-1, all-age data

### ISSUE THREAD

**Issue by leewujung** [OSOceanAcoustics/echopop#229]
Creating this so that we can easily all issues related to processing of fish samples/data that are age-specific or including all ages:
- [x] #220 
- [x] #184 
- [x] #129 
- [x] #91 
- [x] #42

**Comment by rebeccathomas-NOAA** [comment]
FWIW, I've been working through some full-age estimates in Matlab EchoPro, and am 95% sure that even when age-1's are ostensibly included, the age-1s are not all coming through in all of the reports.   That is, I think the age-1 leaks from the "hake" regions are coming through, but the age-1 biomass from the "age-1" regions are not.   Although as best as I can tell, it seems like they are coming through in the "kriged output" reports, but not in the kriged_len_age tables reports.  When I do some digging, I think age-1 biomass from "age-1" regions are not there by the time we finish calculating data.final.table.kriged_Wgt_Len_Age_Matrix_AcoustALL in get_kriged_biomass.m
Is this what you are seeing as well?

**Comment by rebeccathomas-NOAA** [comment]
I followed up on this, looking at 2019.  I was not able to replicate what I saw with 2009 for 2019, nor for 2009 when I went back to it.   2019 looked just fine for age-1's.  In retrospect, I think that the issue was that although the pre-kriging strata for 2009 included age-1 hauls, the post-kriging (geographic) stratification for 2009 did not, which is why the biomass from age-1 region NASC did not seem to flow through into the final kriged age1+ estimate.
